visual-e2e/examples/test/configs/wdio.shared.conf.js

```diff
@@ -1,23 +1,48 @@
+
+
 exports.config = {
     // ====================
     // More information about the configuration is found here https://webdriver.io/docs/configurationfile.html
     // ====================
     runner: 'local',
     specs: [
-        './test/specs/**/*.js'
+        './test/specs/simple.spec.js'
     ],
     maxInstances: 100,
-    logLevel: 'debug',
+    logLevel: 'warn',
```

Comment:
I would put the logs to `silent`, which is the default one for all examples in this repo